### ISSUE TITLE
docs: search loading causes layout shift

### ISSUE THREAD

**Issue by tryonelove** [TanStack/tanstack.com#287]
### Describe the bug

While orama search is loading, it shows a spinner, however, at some point, the search input is hidden which causes layout shift

### Steps to reproduce

Open https://tanstack.com/query/latest/docs/framework/react/overview and look at the search

CPU slowdown can be enabled for more obvious results

### Expected result

No layout shift

### Video

https://github.com/user-attachments/assets/c8b6a820-7760-4672-9851-f660ea96323f

**Comment by NathanBeddoeWebDev** [comment]
Yes!! It pushes it enough that you end up clicking on the wrong docs button if you're even average mouse speed.

Please fix! Chuck a min height on the overarching container or something for the spinner and search input to sit in.

**Comment by tannerlinsley** [comment]
Outdated
